murenn/dtcwt/nn.py

```diff
@@ -20,13 +23,15 @@ def __init__(self, *, J, Q, T, in_channels, padding_mode="symmetric"):
         self.dtcwt = murenn.DTCWT(
             J=J,
             padding_mode=padding_mode,
+            normalize=True,
```

Comment:
isn't `normalize=True` already the default? are you saying there should be no default?